### ISSUE TITLE
Dependency "decode-uri-component": "^0.2.0" contains vulnerability

### ISSUE THREAD

**Issue by danisluk** [sindresorhus/query-string#350]
Unmaintained library decode-uri-component contains quite severe vulnerability.
https://github.com/advisories/GHSA-w573-4hg7-7wgq
Any chance, this could be fixed here possibly by replacing the lib?

**Comment by MasterJuan** [comment]
Also, npm audit fix proposes the below change:
```
Will install **query-string@4.3.4**, which is a breaking change
node_modules/decode-uri-component
  query-string  >=5.0.0
  Depends on vulnerable versions of decode-uri-component
  node_modules/query-string
```
So, actually proposes a downgrade of query-string version.

Instead of using decode-uri-component can you replace it with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURIComponent

**Comment by redonkulus** [comment]
@sindresorhus what do you suggest to mitigate this vulnerability?

**Comment by ryanadhi** [comment]
seems like a fix is on the way on decode-uri-component https://github.com/SamVerschueren/decode-uri-component/issues/6#issuecomment-1333249785

**Comment by taejs** [comment]
@viczhuravlev 
@sindresorhus
@rawle51

Hi
query-string@5 is EOL? 
I just wonder there's any plans to also apply this changes to v5, v6

I can create PR for this

**Comment by sindresorhus** [comment]
> I just wonder there's any plan to also apply this changes to v5, v6

No plans, but it's a patch release, so you should get the latest one there anyways thanks to server.

**Comment by rutanika** [comment]
@viczhuravlev
@sindresorhus
@rawle51

The issue was fixed in decode-uri-component version 0.2.2.
https://github.com/SamVerschueren/decode-uri-component/issues/6
Can you please upgrade?
